src/index.js

```diff
@@ -242,10 +395,10 @@ class ClientAuthOIDC {
    * clear any http-only session cookies.
    */
   logout () {
-    this.clearCurrentUser()
-
     let logoutEndpoint = this.providerEndSessionEndpoint()
 
+    this.clearCurrentCredentials()
```

Comment:
I'm not sure if I would do this before the log out transaction with the IDP succeeds.